src/helpers/3box.ts

```diff
@@ -1,6 +1,6 @@
 import { subgraphRequest } from '@snapshot-labs/snapshot.js/src/utils';
 
-export async function getProfiles(addresses) {
+function get3BoxProfiles(addresses) {

```

Comment:
The method `get3BoxProfiles ` and the method `lookupAddresses` should return the same format like this:
```
{
  "0x123": { name: "Fabien" }
}
```
And for ENS
```
{
  "0x123": "fabien.eth"
}
```
That logic should not be in the `getProfiles` method